src/index.js

```diff
@@ -25,7 +25,7 @@ rtm.on(CLIENT_EVENTS.RTM.RTM_CONNECTION_OPENED, () => {
 const A_WEEK_IN_SECONDS = 604800;
 
 function isEmoji(str) {
-  return /^:[\w-]+:$/.test(str);
+  return /^:[\w-+]+:$/.test(str);
```

Comment:
why? Is this related to this feature?

Reply:
Because, we have emoji 👍  with code (:"+1":)

Reply:
ok, so it is unrelated ;)